src/class.js

```diff
@@ -0,0 +1,50 @@
+export default class Tasks {
+  constructor() {
+    this.list = localStorage.getItem('tasks')
+      ? JSON.parse(localStorage.getItem('tasks'))
+      : [];
+  }
+
+  add(task) {
+    task.index = this.list.length + 1;
+    task.completed = false;
+    this.list.push(task);
+
+    localStorage.setItem('tasks', JSON.stringify(this.list));
+  }
```

Comment:
- [ ] Tasks are being saved to local storage but they are not being displayed on the screen. Please fix this issue.